jobs/v3/snippets/basic_company_sample.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Google Inc. All rights reserved.
```

Comment:
Looks like there is a lot of duplicated code in this directory - could this all be in a single file named `snippets.go`?